bulk_adding/views.py

```diff
@@ -199,6 +201,7 @@ def update_person(self, context, data, person_extra):
                 extra__election=election,
                 role=election.candidate_membership_role,
             ):
+            raise_if_unsafe_to_delete(old_membership)
```

Comment:
I have a slight concern about having code that could bail part of the way through an update resulting in things being in an inconsistent state. I'm not sure I can think of how that might happen at the moment so it's probably fine but it'd be good to revisit this in the future.